addons/mail/models/mail_mail.py

```diff
@@ -336,6 +337,17 @@ def _send_prepare_values(self, partner=None):
         """
         self.ensure_one()
         body = self._send_prepare_body()
+        if partner and 'unfollow_placeholder_start' in body and \
+                self.mail_message_id.model and self.mail_message_id.res_id and \
+                (getattr(self.env[self.mail_message_id.model], '_partner_unfollow_enabled', False) or
+                 any(user._is_internal() for user in partner.user_ids)):
+            unfollow_url = self.env['mail.thread']._notify_get_action_link('unfollow',
+                                                                           model=self.mail_message_id.model,
+                                                                           res_id=self.mail_message_id.res_id,
+                                                                           pid=partner.id)
+            body = re.sub(_UNFOLLOW_REGEX, lambda match: match[1].replace('/mail/unfollow', unfollow_url), body)
```

Comment:
in the end, do we need the regex ? can't we just replace the link `/mail/unfollow` ? 

(same logic as `_replace_local_links`)

Reply:
arf indeed